packages/web/app/routes/PatientRoutes.jsx

```diff
@@ -1,177 +1,132 @@
-              component: DischargeSummaryView,
-              title: (
-                <TranslatedText
-                  stringId="encounter.dischargeSummary.title"
-                  fallback="Discharge Summary"
-                />
-              ),
-            },
-            ...(canAccessMar
-              ? [
-                  {
-                    path: `${PATIENT_PATHS.MAR}/view`,
-                    component: MarView,
-                    title: (
-                      <TranslatedText
-                        stringId="encounter.mar.title"
-                        fallback="Medication admin record"
-                      />
-                    ),
-                    subPaths: [
-                      {
-                        path: `${PATIENT_PATHS.ENCOUNTER}?tab=${ENCOUNTER_TAB_NAMES.MEDICATION}`,
-                        title: (
-                          <TranslatedText
-                            stringId="encounter.medication.title"
-                            fallback="Medication"
-                          />
-                        ),
-                        navigateTo: () =>
-                          navigateToEncounter(encounter.id, {
-                            tab: ENCOUNTER_TAB_NAMES.MEDICATION,
-                          }),
-                      },
-                    ],
-                  },
-                ]
-              : []),
-            {
-              path: `${PATIENT_PATHS.ENCOUNTER}/programs/new`,
-              component: ProgramsView,
-              title: 'New Form',
-            },
-            {
-              path: `${PATIENT_PATHS.LAB_REQUEST}/:modal?`,
-              component: LabRequestView,
-              title: 'Lab Request',
-            },
-            {
-              path: `${PATIENT_PATHS.IMAGING_REQUEST}/:modal?`,
-              component: ImagingRequestView,
-              title: 'Imaging Request',
-            },
-          ],
-        },
-        {
-          path: PATIENT_PATHS.PROGRAM_REGISTRY,
-          component: PatientProgramRegistryView,
-          navigateTo: () => navigateToProgramRegistry(),
-          title: <ProgramRegistryTitle />,
-          routes: [
-            {
-              path: PATIENT_PATHS.PROGRAM_REGISTRY_SURVEY,
-              component: ProgramRegistrySurveyView,
-              title: 'Survey',
-            },
-          ],
-        },
+    },
+    {
+      path: 'programs/new',
+      component: ProgramsView,
+      breadcrumbs: [<Breadcrumb key="new-form" title="New Form" />],
+    },
+    { path: 'referrals/new', component: ReferralsView, breadcrumbs: 'New Referral' },
+    {
+      path: 'encounter/:encounterId/:modal?',
+      component: EncounterView,
+      breadcrumbs: [<EncounterBreadcrumb key="encounter" />],
+    },
+    {
+      path: 'encounter/:encounterId/summary/view',
+      component: DischargeSummaryView,
+      breadcrumbs: [
+        <EncounterBreadcrumb key="encounter" />,
+        <Breadcrumb
```

Comment:
Before we were generating the breadcrumbs by traversing the routes but I think this is way simpler